src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random rnd = new Random();
+        Colors[] colors = Colors.values();
+        return (colors[rnd.nextInt(Colors.values().length)].toString());
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/205386160-4c641b4d-81ff-4abc-b5d5-d9efeaab8b36.png)

```suggestion
        return (colors[rnd.nextInt(Colors.values().length)].name());
```